### PR TITLE
Multiples revisions in md files: 02/03/04/05/06 

### PR DESCRIPTION
**Pull request by qiyanjun**
I have added a list of revisions. 

Besides, I feel the example given in the 3rd paragraph of Section “Deep Learning and Patient Categorization” needs more clarifications. Thank you!

### CODE REVIEW

**Comment by agitter** [comment]
Thanks @qiyanjun.  Did you only move the `Applications of deep learning to longitudinal analysis` section or move it and make modifications?

We have more incoming pull requests than reviewers now, so it will be helpful if authors besides @cgreene  and I can review these edits.  In particular, merging some of the older pull requests will allow us to resolve conflicts in these new ones.  For instance, #559 also modified some of the Discussion.  That's a general comment about reviewing, not a specific request to @qiyanjun.

**Comment by qiyanjun** [comment]
I have only moved the Applications of deep learning to longitudinal analysis section with no modification). I felt the original position of that subsection does not connect well to the ### Challenges and opportunities in patient categorization. 

Please feel free to keep it at the original place though.

**Comment by qiyanjun** [comment]
No problem. Glad to review pull requests if helpful.

File: sections/03_categorize.md [hunk 1]
```diff
@@ -617,10 +647,14 @@ Despite the difficulties and uncertainties, machine learning practitioners (and
...
 prevent harm from discriminatory predictions. To reach their potential in this
-domain, deep learning methods will need to be interpretable. Researchers need to
+domain, deep learning methods will need to be interpretable (more in the discussion section). Researchers need to
 consider the extent to which biases may be learned by the model and whether or
 not a model is sufficiently interpretable to identify bias. We discuss the
 challenge of model interpretability more thoroughly in Discussion.
+<<<<<<< HEAD
+
+=======
+>>>>>>> greenelab/master
```
Comment (dhimmel) [inline sections/03_categorize.md]: Looks like you committed some lines from a partially resolved conflict. Delete these lines?

File: sections/02_intro.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ algorithms have produced comparable or higher accuracy than previous
 best-in-class methods that required years of extensive customization, and
 specialized implementations are now being used at industrial scales.
 
+Deep learning is originated from the field of neural networks.
```
Comment (dhimmel) [inline sections/02_intro.md]: remove "is"

File: sections/02_intro.md [hunk 3]
```diff
@@ -42,7 +43,7 @@ these steps. When sufficient data are available, these methods construct
 features tuned to a specific problem and combine those features into a
 predictor. Recently, hardware improvements and very
 large training datasets have allowed these deep learning techniques to surpass
-other machine learning algorithms for many problems.
+other machine learning algorithms for many important problems [@tag:Krizhevsky2013_nips_cnn @arxiv:1412.5567].
```
Comment (dhimmel) [inline sections/02_intro.md]: new syntax for multiple citations (place semicolon in between)

File: sections/03_categorize.md [hunk 4]
```diff
@@ -401,23 +427,27 @@ has not yet been attempted, it is possible that analogous strategies may be
...
-applied in other domains and may also be adapted to this challenge, e.g. data
+Methods to accomplish more with little high-quality labeled data arose
+in other domains and may also be adapted to this challenge, e.g. data
 programming [@arxiv:1605.07723]. In data programming, noisy automated labeling
-functions are integrated. Numerous writers have described data as the new oil
+functions are integrated.
+
+
+
+Numerous writers have described data as the new oil
```
Comment (dhimmel) [inline sections/03_categorize.md]: "commentators" rather than "writers"?

File: sections/03_categorize.md [hunk 5]
```diff
@@ -645,3 +679,4 @@ LSTMs) [@arxiv:1602.00357]. This may be a particular area of opportunity for
 deep neural networks. The ability to recognize relevant sequences of events from
 a large number of trajectories requires powerful and flexible feature
 construction methods -- an area in which deep neural networks excel.
+>>>>>>> greenelab/master
```
Comment (dhimmel) [inline sections/03_categorize.md]: Remove this line

File: sections/03_categorize.md (hunk 4 above)
Comment (qiyanjun) [reply sections/03_categorize.md]: "commentators" is better.

File: sections/03_categorize.md (hunk 1 above)
Comment (qiyanjun) [reply sections/03_categorize.md]: Yes. Should I do it?

File: sections/03_categorize.md (hunk 5 above)
Comment (qiyanjun) [reply sections/03_categorize.md]: OK

File: sections/03_categorize.md (hunk 1 above)
Comment (agitter) [reply sections/03_categorize.md]: @qiyanjun yes, please remove these extra lines.  GitHub is now showing new conflicts with several files so we'll have to resolve those as well before we can merge this.

File: sections/02_intro.md (hunk 3 above)
Comment (qiyanjun) [reply sections/02_intro.md]: fixed

File: sections/03_categorize.md (hunk 5 above)
Comment (qiyanjun) [reply sections/03_categorize.md]: Trying to fix the error.

Comment (qiyanjun) [reply sections/03_categorize.md]: I have corrected all places according to reviews of @dhimmel.  
(a bit confused by how to update my pull requests when my fork is quite behind the main. not sure if I have done correctly!)

**Comment by dhimmel** [comment]
@qiyanjun are all the actual changes you made part of https://github.com/greenelab/deep-review/pull/636/commits/88eb8f3401fec014c2089773f8c514b610e7f299 and
https://github.com/greenelab/deep-review/pull/636/commits/3adde57fb93c9bef6d094156ddeafc4e40710f2d? If so, I can apply these changes to the latest manuscript version for you?

**Pull request closed without merging**